SpiffWorkflow/SpiffWorkflow/task.py

```diff
@@ -272,6 +272,7 @@ def log_info(self, dct=None):
         extra = dct or {}
         extra.update({
             'workflow': self.workflow.spec.name,
+            'workflow_name': self.workflow.spec.description,
```

Comment:
I would suggest also renaming `workflow` to `workflow_spec`, for consistency with task spec names and descriptions.

Reply:
done